canvas_modules/common-canvas/src/common-properties/controls/textfield/textfield.jsx

```diff
@@ -34,7 +34,7 @@ class TextfieldControl extends React.Component {
 	constructor(props) {
 		super(props);
 		this.reactIntl = props.controller.getReactIntl();
-		this.charLimit = ControlUtils.getCharLimit(props.control, CHARACTER_LIMITS.TEXT_FIELD);
+		this.charLimit = ControlUtils.getCharLimit(props.control, props.controller.getPropertiesConfig().maxLength);
```

Comment:
Might be better to create a new controller method for getting these values so it's clearer